### DISCUSSION TITLE
Plans to include Gemini or ChatGPT vision to transcribe images in notes?

### DISCUSSION THREAD

**Discussion by jhmonroe** [logancyang/obsidian-copilot#464] [Ideas]
If an image of a handwritten note is placed inside a note, currently Copilot does not have the ability to transcribe using Gemini Pro or ChatGPT vision capabilities. Correct?

Any plans to add this?

**Comment by logancyang** [comment]
This is needed for sure, will priortize. Any help is appreciated if people wanna join.
